presto-cli/src/main/java/io/prestosql/cli/FormatUtils.java

```diff
@@ -165,6 +165,24 @@ public static String formatTime(Duration duration)
         return format("%s:%02d", minutes, seconds);
     }
 
+    public static String formatFinalTime(Duration duration)
+    {
+        int totalSeconds = Ints.saturatedCast(duration.roundTo(SECONDS));
+
+        if (totalSeconds >= 60) {
+            int minutes = totalSeconds / 60;
+            int seconds = totalSeconds % 60;
+
+            return format("%s:%02d", minutes, seconds);
+        } else {
```

Comment:
Nit: the "else" is redundant, since the "if" always returns
```java
if (... above cutoff...) {
    return formatTime(...);
}

... format with millis ...